library/crates/BUILD.bazel

```diff
@@ -23,7 +23,79 @@ alias(
 
```

Comment:
Adding `futures`, `grpc`, `protobuf` and `tls_api` crates required for `typedb-client-rust` and `typedb-protocol`